tests/Elastic.CommonSchema.Tests/Specs/868cdfb178bc1959003c62aefed4d5b0c10af4c0/spec.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
A task to fetch latest `./build.sh update-logging-spec` would be amazing 🏅

Reply:
done!